Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SD-2-B.yml

```diff
@@ -20,19 +20,19 @@ title:
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  Risk of leaking production secrets is mitigated by removing any manual interactions during deployment.
+  Risk of leaking production secrets is reduced by removing them from source code files.
 
 #A one sentence description of the activity
 shortDescription:
   Inject secrets dynamically during deployment process from hardened storages and audit all human access to them. 
 
 #A multi-paragraph description of the activity
 longDescription: 
-  Have an automated process to add credentials and secrets appropriate for the target environment to configuration files during the deployment process. This way, developers and deployers do not see or handle those sensitive values.
+  Have an automated process to add credentials and secrets to configuration files during the deployment process. This way, developers and deployers do not see or handle those sensitive values.
```

Comment:
The idea here was to explicitly mention that that the secrets should match the respective stage (e.g. QA, integration, production), do you think it's redundant here?

Reply:
That is the same IMO. We could make it more explicit if you want. Currently did not do this yet.

Reply:
How about "Have an automated process to add credentials and secrets to configuration files during the deployment process to respective stages."?

Reply:
OK, updated.